README.md

```diff
@@ -0,0 +1,189 @@
+# sigstore-installer GitHub Action
+
+This action enables you to install the sigstore tools to be used in your pipeline.
+
+Currently it install the following tools:
+
+1. [cosign](https://github.com/sigstore/cosign)
+2. [rekor-cli](https://github.com/sigstore/rekor)
+
+
+For a quick start guide on the usage of `cosign`, please refer to https://github.com/sigstore/cosign#quick-start.
+
+For available `cosign` releases, see https://github.com/sigstore/cosign/releases.
+For available `rekor-cli` releases, see https://github.com/sigstore/rekor/releases.
+
+## Usage
+
+This action currently supports GitHub-provided Linux, macOS and Windows runners (self-hosted runners may not work).
+
+Add the following entry to your Github workflow YAML file:
+
+```yaml
+uses: sigstore/sigstore-installer@main
+with:
```

Comment:
We could trop the `with` argument, since version pinning is demonstrated by the next two examples, showing that installing sigstore is a simple one-liner.